ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,80 @@
+name: "🐛 Bug Report"
+description: "Submit a bug report to help us improve"
+title: "🐛 Bug Report: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out our bug report form 🙏
+  - type: textarea
+    id: steps-to-reproduce
+    validations:
+      required: true
+    attributes:
+      label: "👟 Reproduction steps"
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      placeholder: "When I ..."
+  - type: textarea
+    id: expected-behavior
+    validations:
+      required: true
+    attributes:
+      label: "👍 Expected behavior"
+      description: "What did you think would happen?"
+      placeholder: "It should ..."
+  - type: textarea
+    id: actual-behavior
+    validations:
+      required: true
+    attributes:
+      label: "👎 Actual Behavior"
+      description: "What did actually happen? Add screenshots, if applicable."
+      placeholder: "It actually ..."
+  - type: dropdown
+    id: php-version
+    attributes:
+      label: "🐘 PHP Version"
+      description: "What version of PHP are you running?"
+      options:
+        - PHP 7.3.x
+        - PHP 7.4.x
+        - PHP 8.0.x
+        - Different version (specify in environment)
```

Comment:
We should also add a dropdown for `library version`

Reply:
I think we should completely avoid this, and have a separate PR to have org level templates instead. The extra maintenance for +30 repos is a waste.

Reply:
Might be a good idea for an HF issue.

Reply:
Oh I didn't know we could have org level templates ? If yes lets do it..